tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -183,6 +145,9 @@ def _impl(ctx):
     else:
         fail("Unreachable")
 
+    host_system_name = "x86_64-apple-macosx"
```

Comment:
Apologies if this is addressed elsewhere, but what about the arm variant? Or is that not supported yet?

Reply:
do you know how this value is actually used? it was hardcoded before so i guess it doesn't entirely matter, but yea I was thinking the same thing

Reply:
It looks like it is now vestigial. https://cs.opensource.google/search?q=lang:java%20host_system_name%20-f:Test.java&sq=&ss=bazel

Reply:
Nice, I can remove this entirely (probably separately) but that might be a breaking change?